chain/chain/src/types.rs

```diff
@@ -797,7 +796,7 @@ pub trait RuntimeAdapter: Send + Sync {
 
 /// The last known / checked height and time when we have processed it.
 /// Required to keep track of skipped blocks and not fallback to produce blocks at lower height.
-#[derive(BorshSerialize, BorshDeserialize, Serialize, Debug, Clone, Default)]
+#[derive(BorshSerialize, BorshDeserialize, Debug, Clone, Default)]
```

Comment:
how do we know that this no longer needs Serialize?

(this is a public struct, so some other project could be using it, right?)

Reply:
This type is used only as an attribute of of `ChainStore` which itself is not serializable.

While, it's technically possible for someone create a new project, just to use this one type with `Serializable`. It doesn't make much sense for anyone to do so,In the future, we may add an optional `serde` support for those structs if someone requests it.  However, our code doesn't need it, so there is no need to have it by default.

```
pub struct ChainStore {
    /// Latest known.
    latest_known: Option<LatestKnown>,
```